Userland/Applications/TextEditor/MainWidget.cpp

```diff
@@ -345,6 +344,11 @@ MainWidget::~MainWidget()
 {
 }
 
+NonnullRefPtr<Core::ConfigFile> MainWidget::get_config_file()
+{
+    return Core::ConfigFile::get_for_app("TextEditor");
+}
```

Comment:
This will reparse the config file instead of reuse it.
Can we avoid parsing it multiple times?
`MainWidget` already caches the config file in `m_config` after first parse.

Reply:
May be with a pattern like this:
```
static RefPtr<Core::ConfigFile> s_config;

RefPtr<Core::ConfigFile> MainWidget::get_config_file()
{
    if (!s_config)
        s_config = Core::ConfigFile::get_for_app("TextEditor");
    return s_config;
}
```
I don't know hot to return ```s_config``` as ```NonnullRefPtr<Core::ConfigFile>```.